### PR TITLE
Update debug_page.dart

### PR DESCRIPTION
**Pull request by MISHRA-TUSHAR**
Updated to Null Safety.
Changed ref.read to context.read for accessing notifier instances.

### CODE REVIEW

File: lib/src/debug_page.dart [hunk 1]
```diff
@@ -3,33 +3,33 @@ import 'package:flutter_ebook_app/src/common/common.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
 class DebugPage extends ConsumerWidget {
-  const DebugPage({super.key});
+  const DebugPage({Key? key}) : super(key: key);
```
Comment (JideGuru) [inline lib/src/debug_page.dart:6]: Any reason for this?

File: lib/src/debug_page.dart [hunk 2]
```diff
@@ -3,33 +3,33 @@ import 'package:flutter_ebook_app/src/common/common.dart';
...
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final isDarkMode = ref.watch(currentAppThemeNotifierProvider).value ==
-        CurrentAppTheme.dark;
+    final isDarkMode = context.read(currentAppThemeNotifierProvider).value == CurrentAppTheme.dark;
```
Comment (JideGuru) [inline lib/src/debug_page.dart:10]: the theme needs to be **watched** not **read**

File: lib/src/debug_page.dart [hunk 3]
```diff
@@ -3,33 +3,33 @@ import 'package:flutter_ebook_app/src/common/common.dart';
...
+
     return Scaffold(
       body: ListView(
         children: <Widget>[
           ListTile(
             title: const Text('Clear Downloads'),
             onTap: () {
-              ref.read(downloadsNotifierProvider.notifier).clearBooks();
+              context.read(downloadsNotifierProvider.notifier).clearBooks();
```
Comment (JideGuru) [inline lib/src/debug_page.dart:18]: Any real benefit to using `context.read` instead of `ref.read `since we already have a `ref`?

**Review by JideGuru — CHANGES_REQUESTED** [review]
Thanks for the PR. Please check the comments I left below

File: lib/src/debug_page.dart (hunk 2 above)
Comment (MISHRA-TUSHAR) [reply lib/src/debug_page.dart:10]: Alright

**Pull request closed without merging**